src/components/common/deleteModal.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { Container, Row, Col, Modal, Form, Button } from "react-bootstrap";
+
+function DeleteModal(props) {
+  const {
+    showModal,
+    handleClose,
+    modalHeading,
+    objectName,
+    deleteFunction,
+  } = props;
+  return (
+    <Modal show={showModal} onHide={handleClose}>
+      <Modal.Body>
+        <Container>
+          <Row className="center-row">
+            <Col xs={12}>
+              <h1 className="modal-heading">Delete {modalHeading}</h1>
+            </Col>
+          </Row>
+          <Row>
+            <Col xs={12}>
+              <div className="modal-delete-container">
+                <p className="modal-delete-text">
+                  Are you absolutely sure? This action can't be undone. This will permanently delete{" "}
+                  <span className="medium-text">
+                    {objectName}
+                  </span>
+                  .
+                </p>
+                <Button
+                  variant=""
```

Comment:
Should we remove this?

Reply:
No, otherwise the Button component defaults to `variant="primary"`

Reply:
And what you want to achieve by leaving empty?
I don't think it's ideal to pass an empty string! Better override somehow maybe?

Reply:
Along with basic colors, padding and margin stuff; bootstrap also provides in a bunch of selectors (&.disabled, &.active, &:hover, &:focus combined with their combinations). Now, we can override each of that selector and all combinations with vanilla CSS; but imo it's better to pass in an empty string rather than achieving same with 15-20 lines of CSS (better to not add in code; than add it in and then add more code to override previous one)

Reply:
Okay, gotcha!
If it was TS, it would've thrown an error doing that!